src/pages/index.jsx

```diff
@@ -355,9 +356,15 @@ const QuickstartsPage = ({ data, location }) => {
             `}
           >
             {Boolean(search) && <SuperTiles />}
-            {filteredQuickstarts.map((pack) => (
-              <QuickstartTile key={pack.id} featured={false} {...pack} />
-            ))}
+
+            {/* Add pagination grid if no search term or category selected */}
+            {isParamPresent(search) || isParamPresent(category) ? (
+              filteredQuickstarts.map((pack) => (
+                <QuickstartTile key={pack.id} featured={false} {...pack} />
```

Comment:
```suggestion
                <QuickstartTile key={pack.id} {...pack} />
```

Did we still need `featured={false}`?

Reply:
You're right. I think that this is left behind. I couldn't find it referenced in the QuickstartTile component